recipes/mkdocs-exporter/recipe.yaml

```diff
@@ -0,0 +1,64 @@
+context:
+  version: 6.2.0
+
+package:
+  name: mkdocs-exporter
+  version: ${{ version }}
+
+source:
+- url: https://pypi.org/packages/source/m/mkdocs-exporter/mkdocs_exporter-${{ version }}.tar.gz
+  sha256: 75fbd757c8fe8302489a48193c37d632dec2d0112d67fe2f72d1c7d80c6fbbfc
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+  - python ${{ python_min }}.*
+  - poetry-core >=1.0.0
+  - pip
+  run:
+  - python >=${{ python_min }}
+  - mkdocs >=1.4
+  - playwright-python >=1.33
+  - beautifulsoup4 >=4.12.2
+  - lxml >=4.9
+  - libsass >=0.22.0
+  - pypdf >=4.2.0
+
+tests:
+- python:
+    imports:
+    - mkdocs_exporter
+    pip_check: true
+    python_version: ${{ python_min }}.*
+
+about:
+  homepage: https://github.com/adrienbrignon/mkdocs-exporter
+  summary: A highly-configurable plugin for MkDocs that exports your pages to PDF files.
+  description: |
+    A highly configurable plugin for MkDocs that exports your pages as PDF documents.
+    Additionally, it can combine your entire documentation into a single PDF file.
+
+    Features:
+
+      🚀 Fast - All PDF documents are generated concurrently!
+      🎨 Customizable - full control over the resulting documents
+          - Combine all your pages into a single PDF document
+          - Create cover pages
+          - Define custom scripts and stylesheets to customize your PDF documents
+          - Add buttons to the top of your pages (example)
+          - Compatible with material and readthedocs themes
+          - Check out sample PDF documents here
+      ⭐ Powerful - it uses a headless browser and some awesome libraries under the hood to generate PDF documents
+          - Playwright to automate browsers
+          - Paged.js polyfills included (provide Paged Media and Generated Content CSS modules)
+          - Sass support included for your stylesheets
```

Comment:
Let's make it a bit shorter. The first two sentences are good enough and people can (should) always navigate to the docs for more info.

```suggestion
```